packages/popup/src/index.js

```diff
@@ -7,6 +7,41 @@ import {SWRConfig} from 'swr'
 
 // import reportWebVitals from './reportWebVitals'
 
+// Fix chrome extension render problem in external screen
+if (
+  // From testing the following conditions seem to indicate that the popup was opened on a secondary monitor
+  chrome &&
```

Comment:
change this to window.chrome if it's a chrome only bug
firefox don't have `window.chrome`